### DISCUSSION TITLE
Can Power BI Report be searchable like OOB SharePoint Search

### DISCUSSION THREAD

**Discussion by tomobak** [microsoft-search/pnp-modern-search#3309] [Q&A] [answered]
Can Power BI reports be searchable with PnP Modern Search like OOB SharePoint Search?
Current OOB Search has "Power BI" vertical (attached image) and you can search Power BI reports

![image](https://github.com/microsoft-search/pnp-modern-search/assets/43676417/2e67b230-7aca-4e15-9dcd-1dd2587a1ef1)

**Comment by kasperbolarsen** [comment, marked as answer]
Hi
I doubt you can search for Power BI. The Data source would have to be Microsoft Search as SharePoint Search only covers SharePoint and OneDrive.

Using the Dev Tools in the browser it looks like the EntityType for PowerBI is "Report"
![image](https://github.com/microsoft-search/pnp-modern-search/assets/20593570/f7d5af46-7f2f-4856-b596-dc43a8d734dc)

In the Graph documentation https://learn.microsoft.com/en-us/graph/api/resources/searchrequest?view=graph-rest-1.0  the EntityType Report is not listed, and Graph Explorer fails if you specify "report" as the EntityType :-(

**Reply by tomobak** [reply]
Thank you for the clarification! I am wondering how they accomplished with OOB Search Power BI vertical with content source as "Power BI"...

![image](https://github.com/microsoft-search/pnp-modern-search/assets/43676417/02aa4dc2-1ae5-420f-a5aa-591434d55193)

**Reply by kasperbolarsen** [reply]
I guess they cheat and don't use the public API. As an example look at the people search in Graph,  it is still in beta!!

**Reply by wobba** [reply]
@tomobak internally Microsoft has access to all search indexes, and you should petition the Graph Search API to expose more.
